pkgs/applications/science/electronics/hal-hardware-analyzer/default.nix

```diff
@@ -1,90 +1,100 @@
 { lib
 , stdenv
+, boost
+, cmake
 , fetchFromGitHub
 , fetchpatch
-, cmake
+, graphviz
+, igraph
+, llvmPackages
 , ninja
 , pkg-config
 , python3Packages
-, boost
-, rapidjson
 , qtbase
 , qtsvg
-, igraph
+, quazip
+, rapidjson
 , spdlog
+, suitesparse
 , wrapQtAppsHook
-, graphviz
-, llvmPackages
 , z3
-, fmt_8
-, suitesparse
 }:
 
 let
-  igraph' = igraph.overrideAttrs (old: rec {
+  igraph' = igraph.overrideAttrs (final: prev: {
```

Comment:
It'd be nice if upstream was aware that there is an igraph version which they don't support.

Reply:
They are aware, see https://github.com/emsec/hal/pull/487

Reply:
> They are aware, see [emsec/hal#487](https://github.com/emsec/hal/pull/487)

Thanks for the link :) @risicle could you mention this in a comment as well?